initial translation proposal for c-1 and c-2 (#58)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-2/50/56/4/translation.txt

//
//事後{0}=
シトリー=Sitri
少々戯れたかっただけなのですか？　しようがないお客様ですね……= So you just wanted to have a little fun? You're helpless, aren't you...

> 困ったお客様です、あまり時間を取らせないで下さいね？= You're such a troubling customer, please don't take up too much time, allright?

Reviewer: ```suggestion
困ったお客様です、あまり時間を取らせないで下さいね？=You are a difficult customer, please don't take too long, okay?
```

Reviewer: `You're` would be fine/better, oops.
